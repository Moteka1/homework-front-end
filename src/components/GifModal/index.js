import React, { Component } from 'react';
import PropTypes from 'prop-types';

export default class GifModal extends Component {
  render() {
    return (
      <div className="GifModal-modal">
        <div className="GifModal-content">
          <span className="close" onClick={this.props.closeModal}>
            &times;
          </span>
          testing modal
        </div>
      </div>
    );
  }
}

GifModal.propTypes = {
  closeModal: PropTypes.func.isRequired
};
