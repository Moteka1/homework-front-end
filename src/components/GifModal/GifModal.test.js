import React from 'react';
import GifModal from './index.js';
import renderer from 'react-test-renderer';

describe('<GifModal />', () => {
  it('matches the snapshot', () => {
    var tree = renderer.create(<GifModal />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
