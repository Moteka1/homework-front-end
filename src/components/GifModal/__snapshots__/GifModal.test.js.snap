// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GifModal /> matches the snapshot 1`] = `
<div
  className="GifModal-modal"
>
  <div
    className="GifModal-content"
  >
    <span
      className="close"
    >
      Ã—
    </span>
    testing modal
  </div>
</div>
`;
